src/commands/sourcemaps/upload.rs

```diff
@@ -47,6 +47,13 @@ pub fn make_command(command: Command) -> Command {
                 .value_parser(validate_distribution)
                 .help("Optional distribution identifier for the sourcemaps."),
         )
+        .arg(
```

Comment:
What's that for? 👀

Reply:
It holds an arbitrary string value stored in the bundle to identify the bundle in the UI. We probably want to pass the SHA1 hash of the CI build here or similar.